src/components/organisms/UserInfo/index.js

```diff
@@ -1,54 +1,57 @@
 import React from "react";
 
+import { useSelector, useDispatch } from "react-redux";
 import styled from "styled-components";
-import PropTypes from "prop-types";
 
+import { userSliceActions } from "../../../modules/slices/userSlice";
 import theme from "../../../theme/theme";
+import { Button, Text } from "../../atoms";
+import { UserHeader } from "../../molecules";
 
-import { Avatar, Button } from "../../atoms";
-import { UserLevel, UserPerformance } from "../../molecules";
+const { colors } = theme;
 
 const Wrapper = styled.div`
   display: flex;
-  width: 100vw;
-  height: 60vh;
   flex-direction: column;
-  justify-content: space-between;
   align-items: center;
 `;
 
-function UserInfo({ profileImage, nickname, level, score }) {
-  const { colors } = theme;
+const HeaderWrapper = styled.div`
+  margin-top: 2rem;
+`;
+
+const ScoreWrapper = styled.div`
+  margin: 10rem 0 30rem 0;
+`;
+
+function UserInfo() {
+  const dispatch = useDispatch();
+  const userInfo = useSelector((state) => state.user.data);
+  const { profileImage, nickname, level, score } = userInfo;
+
+  const handleLoout = () => {
+    dispatch(userSliceActions.logout());
+  };
 
   return (
     <Wrapper>
-      <Avatar src={profileImage} size="lg" />
-      <UserLevel size="xl" margin="3rem" nickname={nickname} level={String(level)} />
-      <UserPerformance size="xl" margin="3rem" target="Total" count={String(score)} />
+      <HeaderWrapper>
+        <UserHeader nickname={nickname} url={profileImage} level={`Lv ${String(level)}`} />
```

Comment:
```suggestion
        <UserHeader nickname={nickname} url={profileImage} level={`Lv ${level}`} />
```
이렇게 해주어도 좋을 것 같습니다.😀
하단 동일 부분은 comment생략하였습니다.~~

Reply:
넵 수정 완료했습니다~!